apps/ledger-live-mobile/src/components/Nft/NftGallery/NftFilterDrawer/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from "react";
+import { useTranslation } from "react-i18next";
+
+import { TrackScreen } from "../../../../analytics";
+import NftFilterSection from "./NftFilterSection";
+import { NftFilterCurrencyItem } from "./NftFilterItem";
+import { BottomDrawer } from "@ledgerhq/native-ui";
+import { NftGalleryChainFiltersState } from "../../../../reducers/types";
+import { NavigatorName, ScreenName } from "../../../../const/navigation";
+import { useNavigation } from "@react-navigation/native";
+
+type Props = {
+  readonly isOpen: boolean;
+  readonly onClose: () => void;
+  readonly toggleFilter: (filter: keyof NftGalleryChainFiltersState) => void;
+  readonly filters: NftGalleryChainFiltersState;
+};
+const NftFilterDraw: FC<Props> = ({ onClose, isOpen, filters, toggleFilter }) => {
+  const { t } = useTranslation();
+  const navigation = useNavigation();
+  return (
+    <BottomDrawer isOpen={isOpen} onClose={onClose}>
```

Comment:
You should use `QueuedDrawer` here :)

Reply:
Using `QueuedDrawer` provided a myriad of problems with it self un-mounting when the NftList re-rendered after filtering. It seems to have lots of side effects that made it hard to work with. Do you have any input on this? It was much easier to use `BottomDrawer` directly and skip side effects.